docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -384,6 +384,9 @@ The following command operators are supported:
 %ROUTE_NAME%
   Name of the route.
 
+%VIRTUAL_CLUSTER_NAME%
+  Name of the matched Virtual Cluster (if any).
+
```

Comment:
This is not applied to tcp traffic. I remember there is a section define some field name are only reasonable in HTTP but not TCP but I cannot recall

Reply:
Changed. Thanks!